searx/static/themes/simple/src/less/search.less

```diff
@@ -380,10 +380,8 @@ html.no-js #clear_search.hide_if_nojs {
 }
 
 .favicon img {
-  height: 1.8rem;
-  width: 1.8rem;
+  height: 1.25rem;
+  width: 1.25rem;
   border-radius: 20%;
-  background-color: var(--color-favicon-background-color);
-  border: 1px solid var(--color-favicon-border-color);
```

Comment:
Not sure its good to remove background color: most of the transparent favicons are made for a light (white) background .. e.g. a black symbol with a transparent background is ok on a light theme but not on a dark theme where the base color in the background is black (what results in a black symbol on a black background).

Reply:
Ah, yeah, I see... seems like there isn't any one perfect solution. I prefer no background slightly more personally but wrt. defaults a white background makes sense.